### DISCUSSION TITLE
Magick.net : how to improve speed when load 1 bit tiff file

### DISCUSSION THREAD

**Discussion by hsilviu** [dlemstra/Magick.NET#1302] [Help] [answered]
Hi, 

I am using C# 2019 and Magick.NET-Q8-AnyCPU version 11.2.0.0 for loading 1bit tif files.
Everything works great but a bit slow with the way I am using it: 

MagickImage mImg = new MagickImage(img.path); // takes about 22 seconds 
mImg ?.Scale(745, 0); // takes about 17 seconds 

The image is 74500x60500 with 2540 dpi (it is not my file and I can not request a different one) :
https://drive.google.com/file/d/1CniKqpE2lmJo6a56GYyauQoKFgK5QjJT/view?usp=share_link

Is there any way to get a better time for processing?

Best regards, 
Silviu

**Comment by dlemstra** [comment, marked as answer]
Those are images with huge dimensions. There is not much that you can do about it. They take a long time to process.

**Reply by hsilviu** [reply]
I know there are huge I just hoped for a way to save some time. 
Thanks for your answer!
